boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,161 @@
+.. _lpcxpresso54114:
+
+NXP LPCXPRESSO54114
+#####################
+
+Overview
+********
+
+The LPCXpresso54114 board has been developed by NXP to enable evaluation
+of and prototyping with the LPC54110 family of MCUs and its low power
+features make it as easy as possible to get started with your project.
+LPCXpresso™ is a low-cost development platform available from NXP supporting
```

Comment:
Should I replace TM with an asterik as it is ? When I replace it with asterik it appears as an asterik in html page generated . Could you please elaborate a bit more ?